docs/conf/atlantic/2024/outing.rst

```diff
@@ -0,0 +1,24 @@
+:template: {{year}}/generic.html
+
+{% if shortcode == '{{ shortcode }}'%}
+:banner: _static/conf/images/headers/group.png
+
+Write the Docs Boat Ride
+========================
+
+As this year's conference is virtual, we encourage you to go on a boat ride or walk in your own beautiful home town.
+
+{% elif shortcode == 'portland'%}
+
+:banner: _static/conf/images/headers/hike.png
+
+Write the Docs Hike
```

Comment:
Hike shows up on the landing page but I'm wondering if we should make it more generic/less physical activity oriented (maybe we can rename to "Outing"?)